### PR TITLE
Updated with addtional information on tmux

### PR DESCRIPTION
**Pull request by jeffmart**
Additional tmux behavior information added to caveats section

### CODE REVIEW

**Review by ian-h-chamberlain — APPROVED** [review]

File: docs/release_notes_128t_installer_2.7.md [hunk 1]
```diff
@@ -23,3 +23,12 @@ sidebar_label: 2.7
 ------
 - **IN-381:** If `tmux` has been installed on the system by another application, the version 2.7 Installer will have a conflict. 
 	- If the `tmux` package is already present on a system, perform `yum remove tmux`. Then use `yum install 128T-installer` to install the new installer. Be careful to _only_ remove `tmux`; removing anything other than `tmux` may result in 128T operational or installation issues.  
+------
+- **IN-398:** An active tmux session/server process prevents `install128t` from running. The following message would be seen in this case:
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_128t_installer_2.7.md:27]: An active tmux session/server process prevents `install128t` from running, and displays the following error:

File: docs/release_notes_128t_installer_2.7.md [hunk 2]
```diff
@@ -23,3 +23,12 @@ sidebar_label: 2.7
...
+   ```
+      Failed to create install128t session
+      No tmux session with name install128t found
+   ```
+   The following `protocol version mismatch (client 8, server 7)` may also be seen in `/var/log/install128t/tmux_wrapper.log`
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_128t_installer_2.7.md:32]: The following message is logged in `var/log/install128t/tmux_wrapper.log`: `protocol version mismatch (client 8, server 7)`.

File: docs/release_notes_128t_installer_2.7.md [hunk 3]
```diff
@@ -23,3 +23,12 @@ sidebar_label: 2.7
...
+
+   _**Corrective Action:**_ terminate all active tmux sessions/server processes and perform the installation/upgrade operation again.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_128t_installer_2.7.md:34]: Terminate all...

File: docs/release_notes_128t_installer_2.7.md (hunk 1 above)
Comment (jeffmart) [reply docs/release_notes_128t_installer_2.7.md]: Updated

File: docs/release_notes_128t_installer_2.7.md (hunk 2 above)
Comment (jeffmart) [reply docs/release_notes_128t_installer_2.7.md]: updated

File: docs/release_notes_128t_installer_2.7.md (hunk 3 above)
Comment (jeffmart) [reply docs/release_notes_128t_installer_2.7.md]: Updated

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**